orc8r/gateway/python/magma/common/redis/containers.py

```diff
@@ -265,6 +290,8 @@ def __setitem__(self, key: str, value: T) -> Any:
         version = self.get_version(key)
         serialized_value = self.serde.serialize(value, version + 1)
         composite_key = self._make_composite_key(key)
+        if self._writeback:
+            self.cache[composite_key] = value
         return self.redis.set(composite_key, serialized_value)
```

Comment:
since you are still setting the value in redis, this is write-through cache. am I missing someting?